app/src/main/java/com/rentminder/MainActivity.kt

```diff
@@ -117,15 +117,20 @@ fun EditBillAmounts() {
     val wifiBillEdited = remember { mutableStateOf(false) }
     var otherBill by remember { mutableStateOf("") }
     val otherBillEdited = remember { mutableStateOf(false) }
+    var totalBill by remember { mutableStateOf("") }
+    val totalBillEdited = remember { mutableStateOf(false) }
+    var totalperpersonBill by remember { mutableStateOf("") }
+    val totalperpersonBillEdited = remember { mutableStateOf(false) }
+
```

Comment:
needed this variables to create the box in front of the total bill and total bill per person. how to create variable was done from Kotlin Best practices
https://kotlinlang.org/docs/coding-conventions.html

Reply:
Variable names should be lowerCamelCase (totalPerPersonBill, for instance).